src/Backups/BackupFactory.h

```diff
@@ -45,6 +45,8 @@ class BackupFactory : boost::noncopyable
         bool azure_attempt_to_create_container = true;
         ReadSettings read_settings;
         WriteSettings write_settings;
+
+        CreateParams cloneForReadBaseBackup(BackupInfo base_backup_info_, String old_password) const;
```

Comment:
The `clone` name looks more appropriate for classes like `IAST` where there are many implementation of the base interface. It seems the name `getCreateParamsForBaseBackup()` is a bit better.

Reply:
OK, adjusted.